src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -0,0 +1,89 @@
+import { Icon, IconButton, makeStyles, Paper } from "@material-ui/core";
+import React from "react";
+
+type iconPosition = "left" | "right";
+type arrowStyle = "upDown" | "rightLeft";
+
+export interface SectionProps {
+    isOpen?: boolean;
+    setOpen?: (open: boolean) => void;
+    children: React.ReactNode;
+    title: React.ReactNode;
+    collapsible?: boolean;
+    elevation?: number;
+    iconPos?: iconPosition;
+    classProps?: {
+        section?: string;
+        header?: string;
+        content?: string;
+    };
+    arrowStyle?: arrowStyle;
+}
+
+export const Section = ({
+    children,
+    title,
+    collapsible,
+    isOpen = true,
+    setOpen = () => {},
+    elevation = 1,
+    iconPos = "right",
+    classProps = {},
+    arrowStyle = "upDown",
+}: SectionProps) => {
+    const classes = useStyles();
+    const leftIcon = iconPos === "left" ? classes.leftIcon : null;
+    const toggle = () => setOpen(!isOpen);
+
+    return (
+        <Paper elevation={elevation} className={`${classes.paper} ${classProps.section}`}>
+            <div
+                className={`${classes.header} ${leftIcon} ${classProps.header} ${isOpen ? "" : classes.noBorder}`}
```

Comment:
If I am not mistaken, when `leftIcon` is null, it will appear "null" in the class list. There are packages for that (classnames), but you can also do this: `_.compact([name1, name2, name3]).join(" ")`